html/comparison.html.haml

```diff
@@ -292,7 +303,7 @@
                   %a#diff-unmanaged-files{"data-toggle" => "modal", "data-target" => "#diff-unmanaged-files"}
                     Diff files
         .row.scope_content.collapse.in
-          .row
+          .row{"ng-show" => "diff.unmanaged_files.only_in1 && diff.unmanaged_files.only_in2"}
```

Comment:
Why is that section only shown if there are file only in 1 and others only in 2? You'd also want to see it if there's only on e of those, no?